_codux/ui-kit/components/components.board.tsx

```diff
@@ -17,6 +17,25 @@ export default createBoard({
         <ComponentWrapper>
             <Kit category="Core Components" title="Components & Elements">
                 <Kit.Section title="Inputs">
+                    <Kit.Item>
+                        <Variant name="Input">
+                            <input className="text-input" value="text input" onChange={() => {}} />
```

Comment:
I'm thinking if we should provide `value` and `onChangeProps` as it means that user won't be able to type anything in the board.
On the other hand these are convenient to have when user drops this input from the add panel.

@alisey what do you think?